src/cpp/shared/scripting/consts.cpp

```diff
@@ -320,6 +320,7 @@ void set_creature_consts(lua_State* L) {
   lua_setconst(L, "CREATURE_BLACK_KNIGHT", CREATURE_BLACK_KNIGHT);
   lua_setconst(L, "CREATURE_CATOBLEBA", CREATURE_CATOBLEBA);
   lua_setconst(L, "CREATURE_TREANT", CREATURE_TREANT);
+  lua_setconst(L, "CREATURE_CYBER_SHADOW_ASSASSIN", CREATURE_CYBER_SHADOW_ASSASSIN);
```

Comment:
Nit: Swap the order of this and the Indigo panther so they'll be in the correct order